backend/src/fun_forecast_backend/weather/weather.py

```diff
@@ -3,6 +3,7 @@
 
 import requests
 from requests import Response
+from decimal import *
```

Comment:
Actually wait, only import that functions you are using. Don’t import *, it’s bad practice